src/shared/components/oncoprint/tabularDownload.ts

```diff
@@ -104,49 +104,122 @@ export function getTabularDownloadData(
                 columnMode === 'sample'
                     ? geneticTrackDatum.sample!
                     : geneticTrackDatum.patient!;
-            oncoprintData.CNA[currentGeneName][id] = '';
-            oncoprintData.MUTATIONS[currentGeneName][id] = '';
-            oncoprintData.MRNA[currentGeneName][id] = '';
-            oncoprintData.PROTEIN[currentGeneName][id] = '';
-            oncoprintData.STRUCTURAL_VARIANT[currentGeneName][id] = '';
+
+            // Only initialize each alteration for sample/patient if it doesn't exist
+            // That can happen when we have multiple OncoPrint tracks of the same gene (using OQL)
```

Comment:
the comment here is somewhat confusing.   i think you mean,   "it may already exist if there are multiple oncoprint tracks of the same gene (e.g. different subtypes)

Reply:
I've adjusted the comment